Resources/Prototypes/_Impstation/Entities/Mobs/Customization/Markings/misc2.yml

```
@@ -0,0 +1,165 @@
+- type: marking
+  id: HumanoidEyebags
+  bodyPart: Head
+  markingCategory: Head
+  speciesRestriction: [Dwarf, Human, SlimePerson, Felinid, Oni, Harpy, Reptilian, Vulpkanin, Arachne, Lamia, Tajaran, IPC]
+  sprites:
+  - sprite: _Impstation/Mobs/Customization/markingsbundle2.rsi
```

YAMLlint (1.35.1)

[error] wrong indentation: expected at least 3

(indentation)